### DISCUSSION TITLE
mantine 6 to mantine 7 about <MantineProvider/>

### DISCUSSION THREAD

**Discussion by LucasBlue** [mantinedev/mantine#4994] [Q&A]
When I upgraded from mantine6 to mantine7, I encountered an issue. In the MantineProvider component, the theme attribute used to accept colors in the following format:

colors: {

    transparent: ["transparent"],

},

However, now it throws an error stating: TS2322: Type '[string]' is not assignable to type 'MantineColorsTuple'.

Source has 1 element(s) but target requires 10.

I believe this issue occurred due to the upgrade to mantine7, but I am unable to find a solution. I am using tailwind in conjunction with Mantine, and my goal is to remove the default background color provided by mantine. Is there anyone who knows why this is happening and can provide a solution?

**Comment by rtivital** [comment]
Set `transparent: Array(10).fill("transparent") as any`
